pkg/common/unittestcommon/utils.go

```diff
@@ -39,6 +39,7 @@ func GetFakeContainerOrchestratorInterface(orchestratorType int) (commonco.COCom
 				"volume-extend": "true",
 				"volume-health": "true",
 				"csi-migration": "true",
+				"file-volume":   "true",
```

Comment:
Don't you also need to add the FSS in pvcsi.yaml?

Reply:
done